nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -428,14 +462,71 @@ void reportInitErrorOnField(Symbol symbol, VisitorState state, Description.Build
                   FIELD_NO_INIT, "@NonNull static field " + fieldName + " not initialized"),
               tree,
               builder,
-              state));
+              state,
+              symbol));
     } else {
       state.reportMatch(
           createErrorDescription(
               new ErrorMessage(FIELD_NO_INIT, "@NonNull field " + fieldName + " not initialized"),
               tree,
               builder,
-              state));
+              state,
+              symbol));
+    }
+  }
+
+  /**
+   * Suggests a type change of an element in a source code that can resolve the error.
+   *
+   * @param state Visitor state.
+   * @param target Target element to alternate it's type.
+   * @param errorMessage Error caused by the target.
+   */
+  public void suggest(VisitorState state, Symbol target, ErrorMessage errorMessage) {
```

Comment:
`suggest` is non-descriptive.  How about `serializeFixSuggestion`?

Reply:
3c22394b1652ba1687e5b120b8d09a53ca8229f5